src/templates/index.html

```diff
@@ -1,22 +1,92 @@
+    .square {
+        height: 70px;
+        width: 70px;
+        background-color: #0b6;
+    }
+</style>
+
+<body class="w3-content" style="max-width:1200px">
+{{if not .}}
+    Unfortunately, you are not logged in now :(<br>
+    <a href="/login">Log in!</a>
+{{else}}
+    <!-- Sidebar/menu -->
+    <nav class="w3-sidebar w3-bar-block w3-white w3-collapse w3-top" style="z-index:3;width:265px" id="mySidebar">
+        <div class="w3-container w3-display-container w3-padding-16">
+            <i onclick="w3_close()" class="fa fa-remove w3-hide-large w3-button w3-display-topright"></i>
+            <h3 class="w3-wide"><b>Life Calendar</b></h3>
+        </div>
+        <div class="w3-padding-64 w3-large w3-text-grey" style="font-weight:bold">
+            <a class="w3-bar-item w3-button w3-border-bottom w3-large" href="#">
+                <img src="https://telegram.org/file/464001801/4/pPObBDJVv-M.32191.png/9963667389a3218249" style="width:26%;">
+                Egor Filatov
+            </a>
+            <a href="/friends" class="w3-bar-item w3-button">Friends</a>
+            <a href="/analyze" class="w3-bar-item w3-button">Analyze</a>
+            <a href="/settings" class="w3-bar-item w3-button">Settings</a>
+        </div>
+        <a href="/logout" class="w3-bar-item w3-button w3-padding">Logout</a>
+    </nav>
+
+    <!-- Top menu on small screens -->
+    <header class="w3-bar w3-top w3-hide-large w3-black w3-large">
+        <div class="w3-bar-item w3-padding-24 w3-wide">Life Calendar</div>
+        <a href="javascript:void(0)" class="w3-bar-item w3-button w3-padding-24 w3-right" onclick="w3_open()"><i class="fa fa-bars"></i></a>
+    </header>
+
+    <!-- Overlay effect when opening sidebar on small screens -->
+    <div class="w3-overlay w3-hide-large" onclick="w3_close()" style="cursor:pointer" title="close side menu" id="myOverlay"></div>
+
+    <!-- !PAGE CONTENT! -->
+    <div class="w3-main w3-row-padding w3-margin-top" style="margin-left:265px">
+
+        <!-- Push down content on small screens -->
+        <div class="w3-hide-large" style="margin-top:83px"></div>
+
+        <!-- Day grid -->
+        <div id="dayButtons">
+            <table cols=7 class="w3-table">
+                <tr v-for="week in getDisplayedWeeks()">
+                    <td v-for="day in week">
+                        <!--<a v-bind:href="getDayUrl(day)">-->
+                        <div class="w3-col square" v-if="isDayFilled(day)"
+                             v-bind:style="{backgroundColor: colorToHex(getFilledDayColor(day))}"></div>
+                        <div class="w3-col square" v-else style="background-color: darkgrey"></div>
+                        <!--</a>-->
+                    </td>
+                </tr>
+            </table>
+        </div>
+    </div>
+
+
+    <script src="https://unpkg.com/vue"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="/scripts/index.js"></script>
```

Comment:
I'm not sure, because I don't which value of `rel` is related to scripts or can be used here. If @FixedOctocat (or someone else) knows which one, then maybe these scripts should be moved

Reply:
@tanya-kta,
1. The fact you don't know which `rel` to use should not affect the decision whether this should be moved to `<head>` or not, of course. I'm asking because I am not closely familiar with the rules of placing scripts on web pages, but I have seen such scripts loaded with the `link` tag inside of `head`, that's why I am proposing this change
2. Answering your question, `rel="preload"` is usually used for js